test/transaction_changes_test.rb

```diff
@@ -126,4 +126,19 @@ def test_serialized_attributes_mutation
     @user.save!
     assert_equal '2.2.2.2', @user.connection_details.first.client_ip
   end
+
+  def test_multiple_changes_back_to_original_value
+    @user.transaction do
+      @user.name = "Dillon"
+      @user.save!
+
+      @user.name = "Dylan"
+      @user.save!
+
+      @user.update!(name: "Daring")
+      @user.update!(name: "Dylan")
```

Comment:
Why bother with more than 2 updates?

Reply:
oh I actually wanted to test `A to B to C back to A` I forgot to remove the second save above